core/src/main/kotlin/plugability/DokkaContext.kt

```diff
@@ -45,6 +48,29 @@ interface DokkaContext {
     }
 }
 
+enum class UnresolvedTypeHandler(val response: (KotlinType?) -> String? = { it?.fqName?.asString() }) {
```

Comment:
You have an enum class here, with parameter `response` defined for each entry. So what is the point of including default value for it?